membrane/client/__init__.py

```diff
@@ -0,0 +1,174 @@
+import json
```

Comment:
maybe we need to go a step further and have membrane.client.flask to reflect the fact that we will want to support multiple application framework in the future.

Reply:
Done. Moved to membrane.client.flask